.changeset/khaki-spies-tan.md

```diff
@@ -0,0 +1,6 @@
+---
+'@interledger/open-payments': minor
+'@interledger/openapi': minor
```

Comment:
I don't think we need to change the `package/openapi`, unless I'm missing something

Reply:
No I think you're right. I was thinking it was the `./openapi/resource-server.yaml` schema changes but that is not the package. It showed up as a changed package when running `pnpm changeset:add` but now I'm realizing that was from a local, uncommitted change. Will remove.